src/app/pages/catalogManagement/catalog-management/catalog-management.html

```
@@ -0,0 +1,250 @@
+
+      <!-- Modal footer -->
+      <div class="flex items-center p-6 space-x-3 rtl:space-x-reverse border-t border-gray-200 rounded-b">
+        <button type="submit"
+        (click)="addCategory()"
+                class="text-white bg-[#2596be] hover:bg-[#20aadc] focus:ring-4 focus:outline-none focus:ring-blue-300 font-medium rounded-lg text-sm px-5 py-2.5 text-center">
+          Add New Category
+        </button>
+      </div>
+    </form>
+  </div>
+</div>
```

HTMLHint (1.5.0)

[error] Tag must be paired, no start tag: [ </div> ]

(tag-pair)